### ISSUE TITLE
Request for RSS

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#825]
We've had a request via twitter for an RSS feed for new lessons.
<img width="305" alt="screen shot 2018-05-07 at 09 00 22" src="https://user-images.githubusercontent.com/646647/39691238-ce41169c-51d4-11e8-9fc2-c170f46a2f07.png">

**Comment by acrymble** [comment]
We have this for blog posts (https://programminghistorian.org/feed.xml) - though I don't know that it's advertised anywhere.

**Comment by mdlincoln** [comment]
We should definitely make this - but I'd love it if someone else took a crack at it :) I'll be glad to review a PR though.

**Comment by walshbr** [comment]
I can take a look as per #828 - I'll play around and report back.

**Comment by walshbr** [comment]
It looks like this actually did used to be part of the behavior of our RSS feed, but [the filtering line](https://github.com/programminghistorian/jekyll/blob/d032ccf998a0d987247de1c70bf8b109489cccd3/feed.xml#L28) is searching for lessons by whether or not they have the default template. Looks like we moved lessons to a different template without updating the feed, so we were missing the lessons and grabbing only a few junk pages. The sorting is also a bit wonky, as I think Jekyll sorts by filename and not date by default. So working on getting the most recent lessons and posts.

**Comment by walshbr** [comment]
Refactored the sorting a bit on the PR based on the fact that we now have translations and will anticipate having more in the future. Will comment further on the PR but once that is merged this can be closed I think.

**Comment by walshbr** [comment]
Updated in #828 - our feed has the most recent posts, original language lessons, and translations now. Though it might be worth thinking about how we could advertise it better on the site, that seems like a separate issue. So closing this request for now as it has been fulfilled.

**Comment by acrymble** [comment]
@walshbr can you let the person who requested it know? Maybe good to advertise it generally on Twitter?

**Comment by walshbr** [comment]
Yep can do @acrymble.
